### ISSUE TITLE
Special accents

### ISSUE THREAD

**Issue by Zirconiumirish** [viromedia/viro#24]
Hi, I'm trying to use ViroText and enter my text between "" but when i'm trying to write "é" or a character like this I have something strange like Ã© .

I save my file in UTF-8 but without success.

Thanks for your help.

Michael

**Comment by VikAdvani** [comment]
Hi Michael,
Thanks for reporting this! We've looked into it and concluded it is a bug with ViroText. I've added this issue as a bug and plan to fix this in our next release. For now, as a work around, if the text is static, you get put a ViroImage with the accented text.

**Comment by Zirconiumirish** [comment]
Hi, no problems I will use the ViroImage for now

Thanks for your work.

**Comment by Zirconiumirish** [comment]
Just one question I have to use image instead of text.

I would like that when I click on a specific image it show me another specific image

Are you able by any chance to show me an simple example in js with 2 random image that would be awesome.
Thanks for your help

Michael

**Comment by achuvm** [comment]
Hi @Zirconiumirish,

See this gist for an example of how to do that:

https://gist.github.com/achuvm/fd336b091413be2ca7b17fa861603ab1

Let us know if you have any other issues!

**Comment by Zirconiumirish** [comment]
Thanks a lot for this example =)

That's exactly what I was looking for but i'm trying to add 2 button at the bottom like "1" and "2" and i can't get it working when I click on "1" to show me the earth and the button "2" show me the moon .

I tried everything ^^ can't get it working

Thanks in advance

Michael

**Comment by achuvm** [comment]
Hi @Zirconiumirish,

I've updated the gist by adding 2 Viro buttons, 1 moon and 1 earth. When you click on the moon, it'll set the top image to the moon, when you tap on the earth, it'll set the image to the earth.

Updated example:
https://gist.github.com/achuvm/fd336b091413be2ca7b17fa861603ab1

Let me know if that works for you. If you're still running into issues, feel free to attach some code and we can attempt to debug it.

Hope that helps,

Andy
